astropy/coordinates/earth.py

```diff
@@ -347,12 +347,13 @@ def of_site(cls, site_name):
         See Also
         --------
         get_site_names : the list of sites that this function can access
-        """
+        """  # noqa
```

Comment:
I'd not do this - we should have coverage of this function eventually.

Reply:
It is complaining that line exceeded 100 characters. Not sure what coverage you are referring to. Can you please clarify? Thanks.

Reply:
OK, never mind!